.github/workflows/test.yml

```diff
@@ -65,25 +65,16 @@ jobs:
         uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.python-version }}
-
-      - name: Get pip cache dir
-        id: pip-cache
-        run: |
-          echo "dir=$(pip cache dir)" >> $GITHUB_OUTPUT
-
-      - name: Cache
-        uses: actions/cache@v3
-        with:
-          path: ${{ steps.pip-cache.outputs.dir }}
-          key:
-            ${{ matrix.python-version }}-v1-${{ hashFiles('**/setup.py') }}-${{ hashFiles('**/tox.ini') }}
-          restore-keys: |
-            ${{ matrix.python-version }}-v1-
+          cache: 'pip'
+          cache-dependency-path: |
+            setup.py
+            tox.ini
+            requirements/*.txt
 
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          python -m pip install -r requirements/tox.txt
+          pip install -r requirements/tox.txt
```

Comment:
Why the difference in invocation?

Reply:
Just because the `pip` command is - to my knowledge - most commonly used without the `python -m` "prefix", and so excluding it makes the code slightly less noisy; the line above, on the other hand, _requires_ having that "prefix" in order to successfully update pip, as far as I know.